### ISSUE TITLE
Checkout: No Shipping Address and Shipping Methods (maybe also no Payment Method)

### ISSUE THREAD

**Issue by tim-breitenstein-it** [magesuite/theme-creativeshop#105]
After Updating MageSuite to from v10.1.0 to v11.0.0 with Creativeshop Theme from v16.6.9 to v17.8.10, maybe even before the update, there are no Shipping Address and Shipping Methods information avaiblable in the Checkout.

![image](https://user-images.githubusercontent.com/65597245/193004589-48079f71-9452-44a8-8191-ed67fc838ff2.png)

Configuration > Sales > Delivery Methods > Free Shipping > Enabled = Yes

Generally the default Magento configuration.

Magento: v2.4.4
MageSuite:  v11.0.0
Creativeshop Theme: v17.8.10 
PHP: v7.4

**Comment by br33f** [comment]
Same issue here.

Magento: 2.4.4
MageSuite: v11.0.1
Creativeshop Theme: v17.9.0

**Comment by drabikowy** [comment]
Hello! Thank you for reporting this issue.

We've checked the issue and it seems that indeed we didn't develop compatibility with magento 2.4.4, as we skpped this version in our project and never used this particular Magento version, jumping right in to work on M2.4.5. 

It seems that the reported issue is related to tags closing in html templates. Cleanup was done in M2.4.4 and probably this Magento version became more strict in this matter. 

With the following commit we were able to fix the reported issue. 
https://github.com/magesuite/theme-creativeshop/commit/570dc1bcaacacac28269f3807a7d8cecc7b14b37 

it's available in theme-creativeshop v17.10.0.

However, we do not know if any other issues might occur between Magento 2.4.4 vs Magesuite 11 as we skipped this Magento update and didn't test living projects in such combination. 

We're working currently on update to Magento 2.4.5 so I would wait for this one + MageSuite v12 with theme-creativeshop v18. We should be able to release these till the end of this year as we're in the after-QA bugfixing phase ;) 

Best regards!

@br33f @tim-breitenstein-it
